src/libraries/System.Console/src/System/ConsolePal.Windows.cs

```diff
@@ -306,20 +306,20 @@ public static bool KeyAvailable
 
         public static ConsoleKeyInfo ReadKey(bool intercept)
         {
-            Interop.InputRecord ir;
+            Interop.INPUT_RECORD ir;
             bool r;
 
             lock (s_readKeySyncObject)
             {
-                if (_cachedInputRecord.eventType == Interop.KEY_EVENT)
+                if (_cachedInputRecord.EventType == Interop.KEY_EVENT)
                 {
                     // We had a previous keystroke with repeated characters.
                     ir = _cachedInputRecord;
-                    if (_cachedInputRecord.keyEvent.repeatCount == 0)
-                        _cachedInputRecord.eventType = -1;
+                    if (_cachedInputRecord.keyEvent.wRepeatCount == 0)
+                        _cachedInputRecord.EventType = 0;
```

Comment:
I've changed the value to `0` from `-1` but it should be OK as all usages of this field are:

```cs
if (_cachedInputRecord.EventType == Interop.KEY_EVENT) // 1
{
}
else // 0, -1 and all other values
{
}
```